### ISSUE TITLE
Replace kernel in host would no longer need `COMMANDLINE_ARGS`

### ISSUE THREAD

**Issue by winstonma** [hydrian/stable-diffusion-webui-rocm#15]
In the [docker-compose.yml](https://github.com/hydrian/stable-diffusion-webui-rocm/blob/main/docker-compose.yml) file, you mentioned that the `Following line is needed to RX 5700`. However, I discovered that the same line is also applicable to my Radeon 680M, which runs on the stock kernel (I tried [version 6.2 kernel](https://www.omgubuntu.co.uk/2023/08/ubuntu-22-04-linux-kernel-6-2) released on early august).

I recently installed the [mainline kernel](https://learnubuntu.com/install-mainline-kernel/) and upgraded to and installed kernel version version 6.1.43, and as a result, I no longer require the `COMMANDLINE_ARGS`. This update has improved the speed of image generation and lower the memory requirement (generate image with higher resolution without out of memory) in stable diffusion. No need to modify anything inside docker.

I think it's best to test it and mention it in [README.md](https://github.com/hydrian/stable-diffusion-webui-rocm/blob/main/README.md) if that works.

**Comment by hydrian** [comment]
What version of stable-diffusion was that. In the past, that command line was needed for SD to detect my RX 6600XT and my RX570

**Comment by winstonma** [comment]
I don't think this flag doesn't required by stable diffusion. I personally guess that flag is required by the [AMD Driver](https://repo.radeon.com/amdgpu-install/) (I guess the flag would pass down from stable diffusion to PyTorch (ROCm) version and then to AMD Driver). After upgrading my AMD Driver I no longer need this flag to get stable diffusion work.

Here is my system
Laptop running AMD 6800U (with 680M, using dedicated 8GB ram)
Ubuntu 22.04.3 with system default 6.2.0-36-generic kernel (I use kernel 6.5+ because the new [AMD P-State EPP Default](https://www.phoronix.com/review/linux65-ryzen-servers) would keep my AMD CPU temperature lower and thus it provides better performance on AMD especially laptop)
Stable Diffusion WebUI 1.6 with PyTorch 2.2 Preview (PyTorch 2.0/2.1 should also work)
AMD Driver 5.7.1

I don't need that flag anymore.

**Comment by winstonma** [comment]
If you don't mind please try to run [my repo](https://github.com/winstonma/stable-diffusion-webui-rocm), which is the clone of your repo with several modification.

EDIT:
To install PyTorch 2.1, please modify the [following line](https://github.com/winstonma/stable-diffusion-webui-rocm/blob/ed07a755ff8d5064b55646316379f4247979f2af/Dockerfile#L55) to
```
RUN pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/rocm5.6
```

It would download the latest AMD driver as well as using the unmodified stable diffusion. It works on my laptop but please see if it work on yours.

BUT please put in mind that I need to use kernel 6.2+ on the host linux system in order to get rid of the flags.

**Comment by winstonma** [comment]
@hydrian May I clarify something

I just checked my `.bashrc` on my host PC (not docker), there is 

```
export HSA_OVERRIDE_GFX_VERSION=10.3.0
```

But after this line is placed on the host PC, I no longer need to put `COMMANDLINE_ARGS` in my docker environment.
